### ISSUE TITLE
Error in docker image with the new GKE version: Kupernetes Plugin: error while loading shared libraries: libapparmor.so.1

### ISSUE THREAD

**Issue by Mistobaan** [GoogleCloudPlatform/continuous-deployment-on-kubernetes#93]
Kupernetes Plugin: error while loading shared libraries: libapparmor.so.1
https://github.com/jenkinsci/docker-jnlp-slave/issues/40

Any suggestions on how to fix it?

**Comment by hugorut** [comment]
@Mistobaan switch the image out for your jnlp-slave to this: https://hub.docker.com/r/dmarell/jenkins-k8s-slave/ it installs necessary libapparmour dependencies

**Comment by Mistobaan** [comment]
@hugorut Thanks that helped. 
one quirk step that I run into is that I did delete the previous kluster but I did not delete the VM disk image, so all the previous configuration was still there. I had to manually change the docker image to dmarell/jenkins-k8s-slave in the configuration page. Following this tutorial helped me fix the problem manually: https://cloud.google.com/solutions/configuring-jenkins-container-engine
